examples/Input/Input.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"qlova.org/seed/client"
+	"qlova.org/seed/client/clientside"
+	"qlova.org/seed/new/app"
+	"qlova.org/seed/new/button"
+	"qlova.org/seed/new/column"
+	"qlova.org/seed/new/row"
+	"qlova.org/seed/new/text"
+	"qlova.org/seed/new/textbox"
+)
+
+func someServerSideFunction(name string) string {
+	message := fmt.Sprintf("Hello %s\n", name)
+	fmt.Printf("Created Message: %s\n", message)
+	return message
+}
+
+func main() {
+
+	Name := &clientside.String{}
+	FullName := &clientside.String{}
+
+	app.New("Input Example",
+		column.New(
+			row.New(
```

Comment:
I wanted to include something like:

```go
seed.If(<FullName  == "">, set.Hidden())
```

here, but I couldn't quite figure out how to create a condition off the clientside value.

Reply:
The following looks like it should work, but doesn't either 🤔
```go
	visible.When(FullName.GetString().Equals(js.NewString("")).Not(), row.New(
		text.New(text.SetStringTo(FullName)),
	)),
```

Reply:
Ah, that's close. All client types implement the client.Bool interface so they are (similar to javascript) can be treated as booleans. So you can achieve the same thing by passing the string directly:
```go 
visible.When(FullName. ...)
```

More complex conditions, can be constructed with the the client/if/* packages.
IE. 
```go
import "qlova.org/seed/client/if/the"
visible.When(the.String(FullName).Equals("SteveMaybe"), ...)
```
or logical and/or not with the all, any & not packages.
```go
import "qlova.org/seed/client/if/any"
visible.When(any.AreTrue(...), ...)
```

You can also change styles and most builtin seed.Option(s) dynamically with the change package.
```go
import "qlova.org/seed/set/change"
change.When(FullName, text.SetSize(rem.New(2))
```